docs/providers/aws/events/apigateway.md

```diff
@@ -191,6 +191,41 @@ functions:
             identityValidationExpression: someRegex
 ```
 
+You can also configure an existing Cognito User Pool as the authorizer, as shown
+in the following example:
+
+```yml
+functions:
+  create:
+    handler: posts.create
+    events:
+      - http:
+          path: posts/create
+          method: post
+          authorizer:
+            arn: arn:aws:cognito-idp:us-east-1:xxx:userpool/us-east-1_ZZZ
```

Comment:
So federated identity pools allows ends up basically being IAM access. So in the method setup in cloudformation we would need to set the  AuthorizationType to AWS_IAM.

So this could be supported